scripts/openshiftci-periodic-tests.sh

```diff
@@ -5,6 +5,41 @@ set -e
 # show commands
 set -x
 
+# check if devfiles are updated and if so, update and test them
+check_devfile() {
+    NOTEQUAL= false
```

Comment:
This is not an affectation, but this is running the `false` command by defining the env variable NOTEQUAL being equals to the empty string (like you would run `DEBUG=true mycommand`)

(shell is very sensitive to spaces :( )

Reply:
Updated!